index.html

```diff
@@ -11957,6 +11957,12 @@ <h2 id="AudioListener">
           <h3>
             Attributes
           </h3>
+          <p>
+            For all of the following <a>AudioParam</a>s, the <a>AudioParam</a>
+            rate is specified by the <dfn>listener <a>AudioParam</a> rate</dfn>
+            which is <a>a-rate</a> when any connected <a>PannerNode</a> is
+            <a>a-rate</a> and is <a>k-rate</a> otherwise.
+          </p>
```

Comment:
But it's not the `PannerNode` that is a-rate or k-rate, it's the node's `panningModel` attribute. Somehow this information got edited out.

Reply:
I think it was clearer before when we explicitly mentioned the `equalpower` panning model and conditioned a-rate on this.